lms/djangoapps/dashboard/tests/test_sysadmin.py

```diff
@@ -120,6 +122,126 @@ def _mkdir(self, path):
         self.addCleanup(shutil.rmtree, path)
 
 
+@unittest.skipUnless(
+    settings.FEATURES.get('ENABLE_SYSADMIN_DASHBOARD'),
+    "ENABLE_SYSADMIN_DASHBOARD not set",
+)
+class TestSysadminEditCourseTabs(SysadminBaseTestCase):
+    """Tests for sysadmin edit_course_tabs tools"""
+
+    def setUp(self):
+        super(TestSysadminEditCourseTabs, self).setUp()
+        self.course = CourseFactory.create(org="test", course="course1", display_name="run1")
+        self._setstaff_login()
+
+    def test_get_course_tabs(self):
+        """Tests all code paths for the sysadmin get_course_tabs method"""
+
+        # test that get_current_tabs works with a valid course_id
+        response = self.client.post(reverse('sysadmin_courses'), {
+            'action': 'get_current_tabs',
+            'course_id': unicode(self.course.id),
+        })
+        self.assertIn("num, type, name", response.content.decode('utf-8'))
+
+        # test that get_current_tabs fails with an invalid course_id
```

Comment:
This looks like it could be split into its own test, `test_get_course_tabs_invalid_course_id` to simplify the individual tests.